SwinjectTests/AssemblerSpec.swift

```diff
@@ -0,0 +1,232 @@
+//
+//  AssemblerSpec.swift
+//  Swinject
+//
+//  Created by mike.owens on 12/9/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+import Swinject
+
+class AssemblerSpec: QuickSpec {
+    override func spec() {
```

Comment:
Function Body Length Violation: Function body should be span 40 lines or less: currently spans 215 lines (function_body_length)